test/kommander_test.go

```diff
@@ -36,9 +37,19 @@ const initialE2EKindConfigPath = "artifacts/kind-config.yaml"
 const dockerUsernameEnv = "DOCKERHUB_ROBOT_USERNAME"
 const dockerPasswordEnv = "DOCKERHUB_ROBOT_TOKEN"
 
+var kommanderBranchFlag = flag.String("kommander-branch", "", "")
+var kommanderRepoRef string
+
 func TestKommanderGroup(t *testing.T) {
 	t.Logf("testing group kommander")
 
+	flag.Parse()
```

Comment:
cobra flags can handle this sort of thing for you, but since this is one. it doesn't matter